### ISSUE TITLE
upgrade to latest version breaks nextcloud

### ISSUE THREAD

**Issue by itswg** [nextcloud/groupfolders#170]
I got this error after attempting to upgrade the plugin.
My nextcloud instance went to maintenance mode and did not come up without intervention.
Running:
nextcloud 13.0.2
php5.6
```
2018/08/08 01:42:00 [error] 29856#100842: *18 FastCGI sent in stderr: "PHP message: PHP Parse error:  syntax error, unexpected ':', expecting ';' or '{' in /usr/local/www/nextcloud/apps/groupfolders/lib/Migration/Version103000Date20180806161724.php on line 19" while reading upstream, client: 192.xx.xx.xx, server: my.domain.com, request: "GET /core/ajax/update.php?requesttoken=x%2FG5JLNOTz3XEXDToNhq5aD6q5lLgNQ%2BfaOa%2BWoNwbc%3D%3AkYfDRvcpHFC6JDP48LQaqpSinq09xJ1SDtbvkys9mPg%3D HTTP/1.1", upstream: "fastcgi://unix:/var/run/php-fpm.socket:", host: "my.domain.com"
```

My fix was to replace commented lines with the lines below:
```        #public function name(): string {
        public function name() {
                return 'rename "group_folders_applicable" to "group_folders_groups"';
        }

#       public function description(): string {
        public function description() {
                return 'rename "group_folders_applicable" to fit within oracle limitations';
```

**Comment by wizdude** [comment]
i was running groupfolders 1.2.2 and line 19 is the "public function name(): string {" line, so there are two lines which required remediation. i changed both the "public function name()" and "public function description()" in the same manner and groupfolders successfully upgraded from 1.2.2 to 1.3.0 with no errors reported.

thanks for the tip.

this instance was running nextcloud 13.0.5 but also php5.6 which i'm guessing is the potential cause.

is this some sort of php7 code incompatible with php5.6 perhaps?

**Comment by joostvkempen** [comment]
I think I've got the same problem: groupfolders 1.2.2 with nextcloud 13.0.5 and PHP5.6, when updating (tried both 1.3.0 and 1.3.2) it keeps processing for a while (1 minute) and then stuck in maintenance mode. I can't deliver logging, because I'm with a Nextcloud service provider. Their advice is to stop using group folders.

In my Group Folders there's over 10Gb of data in 1000's of files. Could there be a problem with the size of the data in group folders? Too much data to process within time-out limits?

**Comment by icewind1991** [comment]
php5.6 compatibility should be fixed with 1.3.3
